crates/re_analytics/src/lib.rs

```diff
@@ -95,6 +95,9 @@ impl Event {
         }
     }
 
+    /// NOTE: due to an earlier snafu, we filter out all properties called
```

Comment:
link to a place that explains the snafu (the release branch?) might be nice. Sounds a little bit like hiding otherwise